includes/class-stats.php

```diff
@@ -28,7 +28,43 @@ private function __construct() {
 	 * Do initialization of all the things needed for stats.
 	 */
 	public function init() {
-		// Do init stuff.
+		$this->initialize_wpdb();
+	}
+
+	/**
+	 * Initialize the alias for the stats table on the wpdb object.
+	 *
+	 * @return void
+	 */
+	private function initialize_wpdb() {
+		global $wpdb;
+		$wpdb->job_manager_stats = $wpdb->prefix . 'job_manager_stats';
+		$wpdb->tables[]          = 'job_manager_stats';
```

Comment:
The reason for including the table name on `tables` is so WP automatically updates the attribute with the proper name with the prefix when switching to another blog on a multisite installation.